_posts/2020-06-05-presentation-rules.md

```diff
@@ -26,33 +26,33 @@ organization
 
 ### Rules
 1. **No Builds**.
-No presentation needs builds, animations or transitions; no exceptions. In
+No presentation needs builds (exposing one bullet point of a list at a time), animations or transitions; no exceptions. In
 business this will invariably go wrong. Your audience will ask to go back a
 slide or two, you end up half way in a build, or worse, with an empty slide.
 None of this is needed. 
 
 2. **Minimum font size 24pt**.
 Yes, not all the words in your mind will fit on that one page. Yes, that
-constraint will actually help you creating a clearer narrative. If it does not
-fit, you are not done clarifying your concise messaging. 
+constraint will actually help you create a clearer narrative. If it does not
+fit, you are not done clarifying your messaging consisely.
```

Comment:
Found `consisely` => `concisely` after spell checking again... Hard to get 100% everything right! Thanks for your help here though!